study_group/models.py

```diff
@@ -29,8 +32,11 @@ def join_group(self, user: User):
         group_member = GroupMember(group_id=self, private_id=user)
         group_member.save()
 
+    def get_group_members_count(self):
+        return self.group_members.count()
+
     def is_group_full(self):
```

Comment:
missing a test for that.

Reply:
`test_join_get_members_count` is the test for this method

Reply:
how is it testing it if it never calls it? it requires a simple test, where you check the results of `get_group_members_count` (either by creating an object and using it, or monkeypatching it) against `self.capacity` and see that the result is what you expected.

Reply:
We need to make sure, in a standalone test, that if someone change `is_group_full` by mistake to be `==` or `<=`, we will notice that immediately.

Reply:
> how is it testing it if it never calls it? it requires a simple test, where you check the results of `get_group_members_count` (either by creating an object and using it, or monkeypatching it) against `self.capacity` and see that the result is what you expected.

I'm not sure if I understand,
Do I need to write a new test for `is_group_full` using `get_group_members_count` with `self.capacity`?

Reply:
There is already a test for `is_group_full`, so there is no need to create another one

Reply:
Is there? I didn't see it. if there is one, you can disregard this comment